src/reportWebVitals.js

```diff
@@ -0,0 +1,15 @@
+const reportWebVitals = (onPerfEntry) => {
```

Comment:
- [ ] Unfortunately, I cannot run your app it is throwing me this error. KIndly fix it:
![Screenshot from 2021-11-22 23-14-45](https://user-images.githubusercontent.com/62705947/142943228-60954e58-ba37-40a8-bee5-c40eaffac5c7.png)

Reply:
Hey @GzimAsani  thanks for your comment, do you think you could give me some guidance here? I just created the project And didn't even write any code, I cannot find a way to move that import to top-level without breaking the app, do you have any suggestions?